### PR TITLE
Estonian names

### PR DESCRIPTION
**Pull request by much-rebel**

### CODE REVIEW

**Comment by Saibamen** [comment]
Please close and reopen this PR to build in Travis again

**Comment by fzaninotto** [comment]
I just restarted the build on Travis

File: src/Faker/Provider/et_EE/Person.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ * @package Faker\Provider\et_EE
+ * @author David Gegelija <code@imdavid.xyz>
+ */
+class Person extends \Faker\Provider\Person
+{
+    /**
+     * @link https://news.err.ee/114745/most-popular-baby-names-of-2014
+     */
+    protected static $firstNameMale = array(
+        'Rasmus', 'Artjom', 'Robin', 'Martin', 'Oliver', 'Romet', 'Sebastian', 'Robert',
```
Comment (fzaninotto) [inline src/Faker/Provider/et_EE/Person.php]: This dataset is too small, please look for a (public) dataset of at least 100 names. Oh, and please order alphabetically, and split line by capital letter/

File: src/Faker/Provider/et_EE/Person.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        'Artur', 'Maksim', 'Markus', 'Marten', 'Karl', 'Kristofer', 'Oskar', 'Daniel',
+        'Hugo', 'Henri', 'Mark', 'Nikita', 'Kirill', 'Sander', 'Kevin', 'Aleksandr', 'Daniil'
+    );
+
+    /**
+     * @link https://news.err.ee/114745/most-popular-baby-names-of-2014
+     */
+    protected static $firstNameFemale = array(
+        'Sofia', 'Eliise', 'Maria', 'Mia', 'Lisandra', 'Mirtel', 'Sandra', 'Emma', 'Laura',
```
Comment (fzaninotto) [inline src/Faker/Provider/et_EE/Person.php]: same

File: src/Faker/Provider/et_EE/Person.php [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        'Darja', 'Arina', 'Milana', 'Alisa', 'Anastasia', 'Lenna', 'Liisa', 'Anna', 'Viktoria',
+        'Elisabeth', 'Polina', 'Marta', 'Aleksandra', 'Marleen', 'Hanna', 'Nora'
+    );
+
+    /**
+     * @link https://en.wikipedia.org/wiki/Category:Estonian-language_surnames
```
Comment (fzaninotto) [inline src/Faker/Provider/et_EE/Person.php]: From what I understand, it's based on the articles on Wikipedia about Estonian people. How is that representative?

**Comment by much-rebel** [comment]
@fzaninotto Hey, I've added more names/surnames from Estonian department of Statistics, but they do provide only top50 for free, extra is paid. Though news.err.ee is one of the most popular news agency in Estonia and I've check wikipedia source for surnames - all of them refers to some real Estonian so it is quite representative.

**Comment by much-rebel** [comment]
Hey @fzaninotto again, so I've asked stat.ee to send me a top100 and they actually did, so I have accumulated all the data I've gathered and updated PR with addition.

**Comment by much-rebel** [comment]
@fzaninotto is this going to be reviewed? if no please close it.

**Comment by pimjansen** [comment]
@much-rebel content wise this looks fine. Could you please cleanup your commit history so we will have a single solid commit with your changes?

**Comment by fzaninotto** [comment]
@pimjansen I don't mind the extra commits, besides you have the ability to squash and merge in GitHub.

**Pull request merged**